mkosi/__init__.py

```diff
@@ -3211,6 +3222,7 @@ def install_distribution(args: MkosiArgs, root: Path, do_run_build_script: bool,
     with mount_cache(args, root):
         install[args.distribution](args, root, do_run_build_script)
 
+    link_rpm_db(root)
```

Comment:
Shouldn't we set this up before invoking dnf? This probably deserves a comment to explain exactly why we do this

Reply:
I added a comment, please see. Overall, I'm not 100% sure that doing this after the fact is appropriate. But if we do it *before*, we'd need to know (or decide ourselves) which location should be used, and I don't think we want to do this, at least yet.